### DISCUSSION TITLE
How to link an image in vitepress?

### DISCUSSION THREAD

**Discussion by maicss** [vuejs/vitepress#3182] [Q&A] [answered]
Repo: https://github.com/maicss/website, with current codes in this repo, I have 3 questions:

1. In preview mode, I can see three images in the main html section, but in the in dev mode, none of them are available. Why?
2. How to insert logo image and `favicon.ico` asset?
3. In dev mode, current codes give warnings: `path public is not necessary`, but delete the public path, in preview mode, all images are broken.

Can anyone helps me with it? Thanks.

**Comment by Zhengqbbb** [comment, marked as answer]
https://vitepress.dev/guide/asset-handling#the-public-directory

> `srcDir: './pages',`
```
mv images/ public/
mv public/ page/
```

```diff
- ![cover.png](../images/cover.webp)
- <img src="../images/cover.webp" width="100" height="100">
+ ![cover.png](/images/cover.webp)
+ <img src="/images/cover.webp" width="100" height="100">
```

**Reply by maicss** [reply]
Thank you very much!!!
